### PR TITLE
go-2443 Get default expense category type from client

### PR DESCRIPTION
**Pull request by andrewmoko**

### CODE REVIEW

**Review by kwburnett — CHANGES_REQUESTED** [review]
Code change looks great! Can we have a test written for this?

File: idemp-rest-test/testing/src/tests/charges.test.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
+import { expenseCategoryApi } from 'src/api/expenseCategories';
```
Comment (kwburnett) [inline idemp-rest-test/testing/src/tests/charges.test.ts]:
```suggestion
import { expenseCategoryApi } from '../api';
```

**Review by kwburnett — CHANGES_REQUESTED** [review]
Will you run `npm run create-index` on this? It'll add the API files correctly so they can just be referenced from the folder.

**Comment by andrewmoko** [comment]
> Will you run `npm run create-index` on this? It'll add the API files correctly so they can just be referenced from the folder.

npm-index

Getting that error. Any clue?

**Comment by kwburnett** [comment]
> > Will you run `npm run create-index` on this? It'll add the API files correctly so they can just be referenced from the folder.
> 
> npm-index
> 
> Getting that error. Any clue?

Have you run `npm install` in the `idemp-rest-test/testing` directory?

**Comment by andrewmoko** [comment]
> > > Will you run `npm run create-index` on this? It'll add the API files correctly so they can just be referenced from the folder.
> > 
> > 
> > npm-index
> > Getting that error. Any clue?
> 
> Have you run `npm install` in the `idemp-rest-test/testing` directory?

that was the issue. thanks

**Review by kwburnett — APPROVED** [review]

**Pull request merged**